internal/bminventory/inventory_v2_handlers.go

```diff
@@ -451,3 +454,95 @@ func (b *bareMetalInventory) V2GetPresignedForClusterCredentials(ctx context.Con
 	}
 	return installer.NewV2GetPresignedForClusterCredentialsOK().WithPayload(&models.Presigned{URL: &url})
 }
+
+func (b *bareMetalInventory) GetInfraEnvDownloadURL(ctx context.Context, params installer.GetInfraEnvDownloadURLParams) middleware.Responder {
+	if b.ImageServiceBaseURL == "" {
+		return common.NewApiError(http.StatusConflict, fmt.Errorf("missing image serviec base URL"))
```

Comment:
I think that conflict is something that usually can be fixed, maybe it should be a bad request?

Reply:
Agree.

Reply:
Any 4xx is usually something that the user can fix. I was pretty stuck on what to return here.

Is there a code for "feature unavailable" that isn't auth related?

Reply:
Maybe I misunderstood.  Doesn't this indicate that the service wasn't properly deployed?

Reply:
for specific configuration this api cannot be used.

Reply:
@filanov right, this is only really relevant when this PR is deployed but we're not using the image service.

Which will be a valid configuration in integration right after this is merged, for example.